fearless/Modules/Staking/StakingRebondSetup/StakingRebondSetupPresenter.swift

```diff
@@ -0,0 +1,217 @@
+import Foundation
+import BigInt
+
+final class StakingRebondSetupPresenter {
+    weak var view: StakingRebondSetupViewProtocol?
+    let wireframe: StakingRebondSetupWireframeProtocol!
+    let interactor: StakingRebondSetupInteractorInputProtocol!
+
+    let balanceViewModelFactory: BalanceViewModelFactoryProtocol
+    let dataValidatingFactory: StakingDataValidatingFactoryProtocol
+    let chain: Chain
+    let logger: LoggerProtocol?
+
+    private var inputAmount: Decimal?
+    private var unbonding: Decimal?
```

Comment:
rebonding? :)

Reply:
Unbonding. It's currently unbonding balance